src/components/AdjustPosition/AdjustPosition.tsx

```diff
@@ -671,7 +713,10 @@ const AdjustPosition: FC<AdjustPositionProps> = ({ collateralBalance, debtBalanc
     approveStatus?.pending,
   ]);
 
-  const buttonDisabled = useMemo(() => transactionState === 'loading' || !canAdjust, [canAdjust, transactionState]);
+  const buttonDisabled = useMemo(
+    () => transactionState === 'loading' || !canAdjust || isOverMaxBorrow,
```

Comment:
isOverMaxBorrow already included in canAdjust

Reply:
fixed